README.markdown

```diff
@@ -24,3 +24,51 @@ Quero ajudar!
 
 [![Bitdeli Badge](https://d2weczhvl823v0.cloudfront.net/rennerocha/dojopuzzles/trend.png)](https://bitdeli.com/free "Bitdeli Badge")
 
+Rodando o projeto!
+----------------------
+
+### Requeriments
+
+   - Python 3.7
+   - Django 3
+
+### Instalando a VirtualEnv
+
+```sh
+$ virtualenv --python=python3 venv
+$ source venv/bin/Activate
+```
+
+### Instalando o projeto
+
+```sh
+$ make install
+```
+
+
+### Rodando o serviço
+```sh
+$ make run
+```
+
+## Criador
```

Comment:
**suggestion**: Even that I am the creator of the project, I prefer not to have my name on this document. Just remove this section. My name will appear together with other contributors.